README.md

```diff
@@ -17,6 +17,7 @@
 	 -i	Set timeout value for project in seconds. Will use 600s (10min) unless specified
 	 -f	Set fail threshold in percentage [0-100], the percentage of test steps that have to pass for the test run to succeed (impacts exit value)
 	 -x	Set device completion threshold in percentage [0-100], the percentage of devices in device group that need to complete for the test run to complete (impacts exit value)
+	 -n	Specify a test_run_id, client will only fetch those results and exit
```

Comment:
I think this functionality differs compared to the other options as this is something to be done *after* the run. This was why I was asking. 
Thinking out loud should the `-h` to be divided to two parts:
  1. Test-run related functionality
  1. Post test-run related functionality